### DISCUSSION TITLE
AI參與藝術遺作製成🎶

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#70] [General]
**新聞大綱：**
1817年，英國皇家愛樂協會委託音貝多芬創作兩首交響曲，其中一首《第十號交響曲》則隨著貝多芬1827年撒手人寰，成了遺作。

早在1988年，音樂學家庫珀就曾蒐集手邊資料，創作出他眼中「最接近貝多芬版」的《第十號交響曲》第一樂章，然而，他的嘗試並沒有獲得同行稱讚，反而招致許多批評。

2019年，奧地利卡拉揚研究所負責人羅德找來多位音樂家、AI專家，打算利用AI模仿貝多芬的風格，替他完成《第十號交響曲》。

整理貝多芬留下來的筆記，並設法將小節按樂章簡單分類，同時也幫助AI「認識貝多芬」。除此之外，還得教會如何「作曲」，讓它熟悉交響曲基本曲式、將簡單旋律發展為完整作品的方法，以及如何搭配樂器等學問。

18 個月譜成《第十號交響曲》，在今年10月9日，德國波昂市舉辦全球首演。但羅德知道這首註定得不到所有人的掌聲──那些主張不該讓AI創作藝術、無法複製人類創意的人，應該都會跳出來抨擊。

「談到藝術，我不認為AI會取代人，反而會成為藝術家拓展新領域、用新方式展現自我的工具。」羅德說道。

**新聞心得：**
我覺得這項嘗試能對於許多偉人的遺作能有很好的延續，讓AI去學習、創作需要投入大量時間跟心力，相信研究團隊跟AI共同創造的這譜曲，能讓喜歡欣賞音樂的人從這感受到貝多芬的精神及新時代的創發！

**新聞來源：**
https://technews.tw/2021/10/10/ai-completes-beethoven-symphony-no10/

**Comment by jim11261314** [comment]
我認為這個最近NFT的發展有些連結
像是NFT目前流行頭像的紙娃娃系統
有沒有可能用AI可以拓展更多圖象的變化

**Comment by qeeeee39** [comment]
我很認同：「人類不會被ai取代」這句话
雖然ai一直在變強，但ai是人類創造的，人類只要好好的跟上潮流學習新科技知識就可以在新的領域創造自己的一片天。

**Comment by ghost** [comment]
雖然是模仿他們的風格 但終歸不是他們創作的 難免遺憾
